test/config/TestConfig.cpp

```diff
@@ -450,3 +454,76 @@ TEST(Config, FleetProvisioningCli)
     ASSERT_STREQ("cli-csr-file", config.fleetProvisioning.csrFile->c_str());
     ASSERT_STREQ("cli-device-key", config.fleetProvisioning.deviceKey->c_str());
 }
+
+TEST(Config, DeviceDefenderCli)
+{
+    constexpr char jsonString[] = R"(
+{
+	"endpoint": "endpoint value",
+	"cert": "cert",
+	"key": "key",
+	"root-ca": "root-ca",
+	"thing-name": "thing-name value",
+    "device-defender": {
+        "enabled": true,
+		"interval": 6
+    }
+})";
+    JsonObject jsonObject(jsonString);
+    JsonView jsonView = jsonObject.View();
+
+    CliArgs cliArgs;
+    cliArgs[PlainConfig::DeviceDefender::CLI_DEVICE_DEFENDER_INTERVAL] = 6;
```

Comment:
How would we know if the DD interval was updated via CLI or Config? Better to have two different unit tests for CLI args and Config args.

Reply:
There is already testing for the interval set by json.